tests/unit/components/pacbio/PacbioRunInfo.spec.js

```diff
@@ -83,4 +85,8 @@ describe('PacbioRunInfo', () => {
             expect(wrapper.find('#comments')).toBeDefined()
         })
     })
+
+    afterEach(() => {
+        wrapper.destroy()
+    })
```

Comment:
How come it needs this?

Reply:
attachDocument needs to be changed to attachTo and the advice is to remove the wrapper so it removes any dom elements that were created. Still trying to work out how to get attachTo working.